app/internal/quotes/daily_quotes.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+import random
+from typing import Optional
+
+from app.database.models import Quote
+
+from sqlalchemy.orm import Session
+
+TOTAL_DAYS = 366
+
+
+def quote_per_day(
+        session: Session, date: date = date.today()
+        ) -> Optional[Quote]:
+    """This function provides a daily quote, relevant to the current
+    day of the year. The quote is randomally selected from a set
+    of quotes matching to the given day"""
+    day_num = date.timetuple().tm_yday
+    quotes = session.query(Quote).filter(
+        Quote.id % TOTAL_DAYS == day_num).all()
```

Comment:
Maybe we should choose at random directly from the database [using the ORM](https://stackoverflow.com/questions/60805/getting-random-row-through-sqlalchemy).